doc/system_requirements.md

```diff
@@ -173,6 +173,65 @@ Covers:
 
 Needs: dsn
 
+#### Creating Adapter Scripts
+`req~creating-adapter-scripts~1`
+
+Users can create Exasol adapter scripts using TDDB.
+
+Optionally they can define a debugger connection. 
+Users can control if the debugger is added by an switch that is independent of the source code. 
+
+Covers:
+
+* [feat~creating-database-objects~1](#creating-database-objects)
+
+Needs: dsn
+
+
+#### Creating Tables
+`req~creating-tables~1`
```

Comment:
Table creation is the same for all dialects, isn't it?

Reply:
yes